src/TouristInfoCards.js

```diff
@@ -0,0 +1,63 @@
+/*Level 3  */
+
+import React from "react";
+
+const TouristInfoCards = () => {
+  // TouristInfoCards
+  return (
+    /* JSX code that holds a div called section  and an article which holds card info */
+    <div className="section">
+      <article className="card">
```

Comment:
This works, but it's a little repetitive, e.g. you're having to say three times that a card has an `h3` element in it, once for each card. Could you try extracting a `Card` component to remove that duplication?

Reply:
Thank you, Daniel. I am thinking to put the info in JSON and map though will this work?